spec/models/event_schedule_spec.rb

```diff
@@ -17,5 +17,47 @@
     it { is_expected.to validate_presence_of(:event) }
     it { is_expected.to validate_presence_of(:room) }
     it { is_expected.to validate_presence_of(:start_time) }
+
+    describe '#start_after_end_hour' do
+      context 'is invalid' do
+        it 'when event schedule start_time is after the conference end_hour, and returns an error message' do
+          conference = create(:conference)
+          event = create(:event, program: conference.program)
+          new_event_schedule = build(:event_schedule, event: event, start_time: conference.start_date + conference.end_hour.hours + 1.hour)
+          expect(new_event_schedule.valid?).to eq false
+          expect(new_event_schedule.errors[:start_time]).to eq ["can't be after the conference end hour (#{conference.end_hour})"]
+        end
+      end
+
+      context 'is valid' do
```

Comment:
the two valid contexts can be merged as to be valid it has to be created before the conference end_date and after the start_date, and basically you have exactly the same text twice.